lib/mandrill_dm/message.rb

```diff
@@ -64,6 +64,17 @@ def html
       mail.html_part ? mail.html_part.body.decoded : mail.body.decoded
     end
 
+    def template
+      return_string_value(:template)
+    end
+
+    # `mail[:template_content].value` returns the variables pre-processed,
+    # `instance_variable_get('@value')` returns them exactly as they were passed in
+    def template_content
+      return [{ name: 'blank', content: '' }] unless mail[:template_content]
```

Comment:
You'll probably throw an API error with this one. Should be nil if there's no content.